core/controllers/contributor_dashboard.py

```diff
@@ -263,18 +264,47 @@ def _content_in_review(self, state_name, content_id, suggestions):
             s.change.content_id == content_id for s in suggestions)
 
 
-class MachineTranslatedStateTextsHandler(base.BaseHandler):
-    """Provide a machine translation of exploration content."""
+class MachineTranslationStateTextsHandler(base.BaseHandler):
+    """Provides a machine translation of exploration content."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
     @acl_decorators.open_access
     def get(self):
         """Handles GET requests. Responds with a mapping from content id to
         translation of form:
+
             dict('translated_texts', dict(str, str|None))
+
         If no translation is found for a given content id, that id is mapped to
         None.
+
+        Params:
+            exp_id: str. The ID of exploration being translated.
+            state_name: str. The name of the exploration state being translated.
+            content_ids: str[]. The content IDs of the texts to be translated.
+            target_language_code: str. The language code of the target
+                translation language.
+
+        Data Response:
+
+            dict(
+                'translated_texts', dict(str, str|None),
```

Comment:
After 'translated_texts' thre should be a colon, not a comma.

Reply:
Fixed.